pkg/types/tags.go

```diff
@@ -11,32 +11,60 @@ import (
 var (
 	CONTAINER        = Flag{Name: "container"}
 	DEBIAN           = Flag{Name: "debian"}
+	DEBIAN9          = Flag{Name: "debian9"}
+	DEBIAN10         = Flag{Name: "debian10"}
 	DEBIAN_LIKE      = Flag{Name: "debian-like"} // nolint: golint
 	REDHAT           = Flag{Name: "redhat"}
 	FEDORA           = Flag{Name: "fedora"}
+	FEDORA30         = Flag{Name: "fedora30"}
+	FEDORA31         = Flag{Name: "fedora31"}
+	FEDORA32         = Flag{Name: "fedora32"}
 	REDHAT_LIKE      = Flag{Name: "redhat-like"} // nolint: golint
 	AMAZON_LINUX     = Flag{Name: "amazonLinux"} // nolint: golint
 	RHEL             = Flag{Name: "rhel"}
+	RHEL7            = Flag{Name: "rhel7"}
+	RHEL8            = Flag{Name: "rhel8"}
 	CENTOS           = Flag{Name: "centos"}
+	CENTOS7          = Flag{Name: "centos7"}
+	CENTOS8          = Flag{Name: "centos8"}
 	UBUNTU           = Flag{Name: "ubuntu"}
+	UBUNTU18         = Flag{Name: "ubuntu18"}
+	UBUNTU20         = Flag{Name: "ubuntu20"}
+	UBUNTU16         = Flag{Name: "ubuntu16"}
 	AWS              = Flag{Name: "aws"}
 	PHOTON           = Flag{Name: "photon"}
+	PHOTON2          = Flag{Name: "photon2"}
+	PHOTON3          = Flag{Name: "photon3"}
 	VMWARE           = Flag{Name: "vmware"}
 	NOT_PHOTON       = Flag{Name: "!photon", Negates: []Flag{PHOTON}}            // nolint: golint
+	NOT_PHOTON2      = Flag{Name: "!photon2", Negates: []Flag{PHOTON2}}          // nolint: golint
+	NOT_PHOTON3      = Flag{Name: "!photon3", Negates: []Flag{PHOTON3}}          // nolint: golint
 	NOT_CONTAINER    = Flag{Name: "!container", Negates: []Flag{CONTAINER}}      // nolint: golint
 	NOT_FEDORA       = Flag{Name: "!fedora", Negates: []Flag{FEDORA}}            // nolint: golint
+	NOT_FEDORA30     = Flag{Name: "!fedora30", Negates: []Flag{FEDORA30}}        // nolint: golint
+	NOT_FEDORA31     = Flag{Name: "!fedora31", Negates: []Flag{FEDORA31}}        // nolint: golint
+	NOT_FEDORA32     = Flag{Name: "!fedora32", Negates: []Flag{FEDORA32}}        // nolint: golint
```

Comment:
@moshloop should the main `NOT` OS tag negates all the subsequent versions, i.e, `NOT_FEDORA` will negate `FEDORA, FEDORA30, FEDORA31, FEDORA32` ?